.github/workflows/deploy_bioscrape.yml

```diff
@@ -0,0 +1,31 @@
+name: deploy_bioscrape
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]  
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
```

Comment:
We can increase this number for faster runtime, right?

Reply:
yes, I would guess so.

Reply:
```
By default, GitHub will maximize the number of jobs run in parallel depending on runner availability. 
```
We can remove the `max-parallel` argument then, it seems.